lambda/translprim.ml

```diff
@@ -484,51 +484,51 @@ let lookup_primitive loc ~poly_mode ~poly_sort pos p =
         3)
     | "%array_safe_get_indexed_by_int64#" ->
       Primitive
-        ((Parrayrefs (gen_array_ref_kind mode, Punboxed_int_index Pint64, Mutable)), 2)
+        ((Parrayrefs (gen_array_ref_kind mode, Punboxed_int_index Unboxed_int64, Mutable)), 2)
```

Comment:
There are some lines over 80 cols here but let's leave these for the moment, or they will conflict badly with one of the other array PRs in flight currently.